project/src/main/webapp/header.html

```diff
@@ -26,7 +26,9 @@
   </head>
   <body>
     <nav class="navbar navbar-expand-lg navbar-light">
-      <a class="navbar-brand" href="events-feed.html">Local Event Organizer</a>
+      <a class="navbar-brand" href="events-feed.html">
+        <img id="logo" src="assets/lion.jpg">
```

Comment:
make sure that the jpg is added to git (I dont see in this PR)

Reply:
It's added in `aug26`, I forgot to do it here but there's a newer version once that branch is merged